### DISCUSSION TITLE
Ambien light is not working on bevy 0.13.1

### DISCUSSION THREAD

**Discussion by elVengador** [bevyengine/bevy#12800] [Q&A]
Hi, Im learing bevy, I added an ambien light but is not working
```rust
fn main() {
    App::new()
        // Bevy built-ins.
        .insert_resource(ClearColor(Color::rgb(0.1, 0.0, 0.15)))
        .insert_resource(AmbientLight { // <-- here is my light 💡
            color: Color::default(),
            brightness: 0.75,
        })
        .add_plugins(DefaultPlugins)
        // User defined plugins.
        .add_plugins(MovementPlugin)
        .add_plugins(DebugPlugin)
        .add_plugins(SpaceshipPlugin)
        .add_plugins(CameraPlugin)
        .run();
}
```
<img width="1291" alt="image" src="https://github.com/bevyengine/bevy/assets/27080940/80d4ab55-b62c-401e-84fb-9824b34a1e9c">

but when I change the bevy version to `0.12.0`, my Ambien light is owrking correctly
<img width="1289" alt="image" src="https://github.com/bevyengine/bevy/assets/27080940/be1101bd-3b67-4d14-afb9-0ef79a46be4b">

Also I cheked How the AmbienLight is set up in this example: https://bevyengine.org/examples/3D%20Rendering/lighting/, but I dont understand why it dosent work with Bevy 0.13.1

**Comment by JMS55** [comment]
Add it _after_ you add DefaultPlugins. DefaultPlugins will insert a new ambient light and overwrite the one you added.

**Reply by DGriffin91** [reply]
This is incorrect. You can set `AmbientLight` before `DefaultPlugins` and it will not be overridden. We use `init_resource` [here](https://github.com/bevyengine/bevy/blob/v0.13.1/crates/bevy_pbr/src/lib.rs#L260) "If the Resource already exists, nothing happens."

Also verified with the 3d_scene example.

**Reply by JMS55** [reply]
Apologies, looks like I was wrong 😅, and this was fixed at some point. It's likely the brightness is the issue, then.

**Comment by NthTensor** [comment]
Also, that may not be very bright given the new exposure settings. The units of `brightness` are in Candella per Square Meter, and the default value is 80. See related issue https://github.com/bevyengine/bevy/issues/11933.
